docs/references/security/rust-canister-development-security-best-practices.md

```diff
@@ -202,27 +202,29 @@ For example, consider the following Motoko code:
 
 The first message that is executed here are lines 2-3, until the inter-canister call is made using the `await` syntax (orange box). The second message executes lines 3-5: when the inter-canister call returns (blue box). We call this part the _callback_ of the inter-canister call. The two messages involved in this example will always be scheduled sequentially. 
 
-**Property 3**: Messages from interleaving calls have no reliable execution ordering.
+**Property 3**: Successfully delivered requests are received in the order in which they were sent. In particular, if a canister A sends `m1` and `m2` to canister B in that order, then, if both are accepted, `m1` is executed before `m2`. 
```

Comment:
It would be nice to extend this with a disclaimer that this is true for the messages _only_ but not the responses.

To quote Alin:
> If m1 and m2 both finish execution within a round (i.e. they're not "long messages" whose execution happens across multiple rounds; and they don't make any downstream calls) then the responses would likely be received in the same order. But it's not a general guarantee. Nor something that should be relied upon.

Reply:
Yes that makes sense. Added this: 

> Note that this property only gives a guarantee on when the messages are executed, but there is no guarantee on the ordering of the responses received.